demisto_sdk/commands/validate/validate_manager.py

```diff
@@ -403,6 +404,21 @@ def run_validation_on_generic_entities(self, dir_path, pack_error_ignore_list):
 
         return all(package_entities_validation_results)
 
+    def is_valid_pack_name(self, file_path, old_file_path):
+        if old_file_path:
+            old_file_path_posix = Path(old_file_path)
+            file_path_posix = Path(file_path)
+            if file_path_posix.parts[0] == old_file_path_posix.parts[0] == 'Packs' and \
```

Comment:
do we not chfeck it if not running relatively to `Packs`?

Reply:
The path will always be relative to the `git` path.